cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -169,10 +241,10 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     sampleAllIndexesAndWait()
 
     profileQuery(
-      title = "Use index with `STARTS WITH`",
+      title = "Prefix search using `STARTS WITH` (single-property index)",
```

Comment:
Do we need the suffix `(single-property index)` when that is still the most common case and effectively implicit?

Reply:
I think so - we need some mechanism to make this very explicit and clear to the reader. Maybe a table or sub-TOC would be better - but not with the current awful levels of nesting. Perhaps there is something that can be done in the new Cypher-only manual. Leaving it to Docs Team...

Reply:
Agree about making it explicit, agree that the suffix is distracting. Let's find another way, but not in this PR.